runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/EmoteClue.java

```diff
@@ -248,6 +254,17 @@ public void makeOverlayHint(PanelComponent panelComponent, ClueScrollPlugin plug
 
 		if (itemRequirements.length > 0)
 		{
+			Client client = plugin.getClient();
+			client.runScript(ScriptID.WATSON_STASH_UNIT_CHECK, stashUnit.getObjectId(), 0, 0, 0);
```

Comment:
should this really be called every single frame?

Reply:
Adam said it was fine as it would only be ran when a player has an emote clue:
https://discordapp.com/channels/301497432909414422/419891709883973642/580553043737575455